lib/libc/stdio/lib_lowoutstream.c

```diff
@@ -76,6 +79,11 @@
 static void lowoutstream_putc(FAR struct lib_outstream_s *this, int ch)
 {
 	DEBUGASSERT(this);
+#ifdef __KERNEL__
+#ifdef CONFIG_CRASH_LOG_SAVE
+	crash_log_to_buffer(ch);
```

Comment:
1. This feature covers the storage of all logs output in lldbg as well as crash logs
2. This feature supports log saving to fs with compress.
 
How about module naming like ll_log_dump?